dspace-rest/src/main/webapp/WEB-INF/security-applicationContext.xml

```diff
@@ -38,7 +38,7 @@
         <property name="authenticationManager" ref="dspaceAuthenticationManager"/>
         <!-- Ensures that after login no redirect is made to the home page, the rest will return a 200 status code -->
         <property name="authenticationSuccessHandler" ref="org.dspace.rest.authentication.NoRedirectAuthenticationLoginSuccessHandler"/>
-        <property name="usernameParameter" value="email"/>
+        <property name="usernameParameter" value="user"/>
```

Comment:
To ensure we have no breaking changes between 6.0 and 6.1, I think we need to revert this back to `email` (and make appropriate changes to the other code).  I agree with the idea to switch this to `user` for 7.0, but as that's a breaking change, we should only do that in a major release.